specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -819,6 +854,30 @@
           "type": "string"
         }
       }
+    },
+    "AcsIncomingCallCustomContext": {
+      "description": "Custom Context of Incoming Call",
+      "type": "object",
+      "properties": {
+        "customHeaders": {
+          "type": "string",
```

Comment:
Are we sure this is not meant to be a dictionary like sipHeaders and voipHeaders?

Reply:
yes